run-hlt-validation

```diff
@@ -39,6 +39,12 @@ cd HLTrigger/Configuration/test/
 ls
 ./runAll.csh IB | tee -a $WORKSPACE/results/runIB.log
 ./examLogs.csh | tee -a $WORKSPACE/results/examLogs.out
+which hltPhase2UpgradeIntegrationTests || true
+if [ $? -eq 0 ]; then
```

Comment:
@mmusich , I think this will always pass due to `|| true` so `$?` will always be `0`. I think you can just do
```
if which hltPhase2UpgradeIntegrationTests 2>/dev/null ; then
```